### PR TITLE
Create monique-umphrey-hcc-interview

### PR DESCRIPTION
**Pull request by Oshratn**
Interview with Monique Umphrey president of the Houston Community College.

### CODE REVIEW

File: content/blog/monique-umphrey-hcc-interview [hunk 1]
```diff
@@ -0,0 +1,74 @@
+# Moving the needle on inclusive language, with Dr. Monique Umphrey from Houston Community College
+ 
+Welcome back to our series **Moving the needle on inclusive language**. In this series, we talk to leaders in the cloud-native space as they share the inclusive naming
```
Comment (xmulligan) [inline content/blog/monique-umphrey-hcc-interview:3]:
```suggestion
Welcome back to our series **Moving the needle on inclusive language**. In this series, we talk to leaders in the cloud native space as they share the inclusive naming
```

File: content/blog/monique-umphrey-hcc-interview [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+ 
+[Dr. Monique](https://www.linkedin.com/in/moniquewilson/) was recently named one of the [top 50 most powerful women in technology](http://top50tech.org/2020/). She 
+became president of Houston Community College in 2019. She was not a typical candidate who had climbed the ranks in academia. Rather, she served as a software developer
+for the first 14 years of her career. She spent the next 10 years leading software development projects. During this time she also taught as an adjunct instructor at a
+community college. She has always been a fan of the underdog and the underrepresented. Working at a community college allowed her to become part of the solution, by
+cultivating more diverse IT talent. She achieves this by increasing awareness about technical career paths for underrepresented populations and ensuring they have viable
+pathways into careers with family sustaining wages.
+ 
+**Please describe a situation where biased language affected you.**
+My freshman year, I was started at a selective R1 research institution. It was not a pleasant experience at all, it was actually hostile. It was not nurturing, it was not
```
Comment (xmulligan) [inline content/blog/monique-umphrey-hcc-interview:14]:
```suggestion
My freshman year, I was at a selective R1 research institution. It was not a pleasant experience at all, it was actually hostile. It was not nurturing, it was not
```

File: content/blog/monique-umphrey-hcc-interview [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+welcoming. I felt like I was being treated like a research subject and it felt like they did not expect me to succeed.
+
+I was lucky that I had people who cared about me outside of that environment. They were determined to show me that I had something inside of me that was worthy of
+cultivating and that I was college material. So I was able to push on. I saw students that didn't have similar support quit their college journey. 
+
+I saw the power of negative language. I saw the power of negative reinforcement and stereotypes. I experienced how powerful language is and how it can be used in a negative
+way. So it was really important to me to create a different type of academic environment that was more nurturing.
```
Comment (xmulligan) [inline content/blog/monique-umphrey-hcc-interview:21]:
```suggestion
way. I realized it is really important to me to create a different type of academic environment that is more nurturing. 
```

**Comment by xmulligan** [comment]
@Oshratn same comment as before, this one also needs a header like
The only other part is you need to add a header at the top like

---
title: "Making Inclusion Part of Your Business is Better for Business. Period."
date: 2021-06-01
slug: inclusive-good-for-business
author: Wendy Rentschler, Shahed Alqadi
---

File: content/blog/monique-umphrey-hcc-interview.md [hunk 4]
```diff
@@ -1,3 +1,8 @@
+title: "Moving the needle on inclusive language. Monique Umphrey"
+date: 2021-08-01
+slug: inclusive-language-inititives-hcc
+author: Oshrat Nir, Shahed Alqadi
```
Comment (xmulligan) [inline content/blog/monique-umphrey-hcc-interview.md:4]:
```suggestion
---
title: "Moving the needle on inclusive language. Monique Umphrey"
date: 2021-08-01
slug: inclusive-language-inititives-hcc
author: Oshrat Nir, Shahed Alqadi
---
```

Comment (xmulligan) [inline content/blog/monique-umphrey-hcc-interview.md:4]:
```suggestion
---
title: "Moving the needle on inclusive language. Monique Umphrey"
date: 2021-08-01
slug: inclusive-language-inititives-hcc
author: Oshrat Nir, Shahed Alqadi
---
```

**Comment by Oshratn** [comment]
Ready to be published on August 25th 2021

**Review by celestehorgan — APPROVED** [review]

File: content/blog/monique-umphrey-hcc-interview.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
+---
+---
+title: "Moving the needle on inclusive language. Monique Umphrey"
+date: 2021-08-25
+slug: inclusive-language-inititives-hcc
+author: Oshrat Nir, Shahed Alqadi
+---
```
Comment (xmulligan) [inline content/blog/monique-umphrey-hcc-interview.md:7]:
```suggestion
```

File: content/blog/monique-umphrey-hcc-interview.md [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
```
Comment (xmulligan) [inline content/blog/monique-umphrey-hcc-interview.md:1]:
```suggestion
```

**Pull request merged**